modules/pensions/lib/pensions/pdf_fill/va21p527ez.rb

```diff
@@ -223,18 +271,10 @@ class Va21p527ez < ::PdfFill::Forms::FormBase
         },
         # 3d
         'serviceNumber' => {
-          limit: 12,
```

Comment:
These items don't overflow anyways, and it was causing issues with "Other Name" to not show line numbers